test/unit/elastic_search_test.rb

```diff
@@ -0,0 +1,286 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+class ElasticSearchTest < MiniTest::Test
+
+  extend Minitest::Spec::DSL
+  include LogWrapper
+
+  def setup
+    @es_model = ElasticSearch.new
+    @mintest_mock1 = MiniTest::Mock.new
+    @mintest_mock2 = MiniTest::Mock.new
+    @agg_hash = {"language" => {:terms => {:field => "primary_language", :size => 100, :order => {:_key => "asc"}}},
```

Comment:
# models the format of the ES query terms, the structure we'll be sending our search requests in